Include/dictobject.h

```diff
@@ -58,13 +58,15 @@ PyAPI_FUNC(PyObject *) PyDict_GetItemString(PyObject *dp, const char *key);
 PyAPI_FUNC(int) PyDict_SetItemString(PyObject *dp, const char *key, PyObject *item);
 PyAPI_FUNC(int) PyDict_DelItemString(PyObject *dp, const char *key);
 
+#if !defined(Py_LIMITED_API) || Py_LIMITED_API+0 >= 0x030A0000
```

Comment:
Are you sure about the version number? It looks like Python 3.10.

Reply:
Good catch. I noticed this immediately after creating the PR, but missed the `-a` option in `git commit`, so my forced push did not fix anything.